yeat/cli/config.py

```diff
@@ -184,3 +199,15 @@ def check_canu_required_params(self):
             raise ValueError(
                 "Canu requires at least 4 avaliable cores; increase `--threads` to 4 or more"
             )
+
+    def __members(self):
+        return (self.label,)
+
+    def __eq__(self, other):
+        if type(other) is type(self):
+            return self.__members() == other.__members()
+        else:
+            return False
+
+    def __hash__(self):
+        return hash(self.__members())
```

Comment:
Roger that. It looks like you based this code on an example incorporating multiple values into the hash function. Since you're only hashing a single value, you could probably simplify with something like this.

```python

def __eq__(self, other):
    return hash(self) == hash(other)

def __hash__(self):
    return hash(self.label)
```

The type check is subsumed in the astronomically small possibility that an object of a different data type will hash to the same numerical value.